pkg/config/node.go

```diff
@@ -261,9 +263,9 @@ func GetConfig(kubeconfigPath, clusterConfigPath, resolvConfPath string, apiVip
 	return node, err
 }
 
-func getSortedBackends(kubeconfigPath string) (backends []Backend, err error) {
+func getSortedBackends(kubeconfigPath string, kubeApiServerUrl string) (backends []Backend, err error) {
```

Comment:
Nit: If there isn't presently any other code that would break due to a change in the order here, I'd rather see the kubeApiServerUrl param come first, to match the order of BuildConfigFromFlags. It's a small nit, but I had to say it. :)

Reply:
Found another solution, I deleted the 'kubeApiServerUrl' parameter. :-)